django-stubs/utils/decorators.pyi

```diff
@@ -1,15 +1,19 @@
 from collections.abc import Callable, Iterable
 from typing import TypeVar
 
+from django.http.response import HttpResponseBase
 from django.utils.deprecation import MiddlewareMixin
 from django.views.generic.base import View
+from typing_extensions import TypeAlias
 
 _T = TypeVar("_T", bound=View | Callable)  # Any callable
```

Comment:
Perhaps we could define generics for the `Callable` here too, while we're at it?

```suggestion
_T = TypeVar("_T", bound=View | Callable[..., Any])
```

Reply:
Nit: let's please not name it `_T`, since it is bound to `View` it can be `_ViewT` or something